Dalamud/EntryPoint.cs

```diff
@@ -56,9 +56,9 @@ public sealed class EntryPoint : IEntryPoint {
 
         private (Logger logger, LoggingLevelSwitch levelSwitch) NewLogger(string baseDirectory) {
 #if DEBUG
-            var logPath = Path.Combine(baseDirectory, "dalamud.txt");
+            var logPath = Path.Combine(baseDirectory, "dalamud.log");
 #else
-            var logPath = Path.Combine(baseDirectory, "..", "..", "dalamud.txt");
+            var logPath = Path.Combine(baseDirectory, "..", "..", "dalamud.log");
```

Comment:
can you add that extra ".." while you're at it

Reply:
sure, done :)